### PR TITLE
Fix "02_autoupgrade" in install_upgrade campaign

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x 
| Description?  | Fix two selectors in the front office( Home page) and correct the waitForExistAndClick function in 02_autoupgrade
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | npm run install-upgrade-test -- --URL=RCURL --DIR=DOWNLOADPATH --URLLASTSTABLEVERSION=STABLEPSURL --RCTARGET=RCTARGET --FILENAME=prestashop_1.7.6.0.zip --DB_SERVER=SERVER --DB_PASSWD=PWD --DB_USER=USR --LANGUAGE=en --COUNTRY=france 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14356)

### CODE REVIEW

File: tests/E2E/test/campaigns/install_upgrade/02_autoupgrade.js [hunk 1]
```diff
@@ -68,11 +68,7 @@ scenario('The shop installation', () => {
     test('should set the Number of the version you want to upgrade to', () => client.waitAndSetValue(ModulePage.version_number, global.filename.replace(".zip", "")));
     test('should click on "save" button', () => client.waitForExistAndClick(ModulePage.save_button));
     test('should verify the success message', () => client.waitForVisibleElement(ModulePage.save_message));
-    test('should click on "Upgrade PrestaShop now!" button', () => {
-      return promise
-        .then(() => client.scrollTo(ModulePage.upgrade_block))
-        .then(() => client.waitForExistAndClick(ModulePage.upgrade_button, 50, 7000));
-    });
+    test('should click on "Upgrade PrestaShop now!" button', () => client.waitForExistAndClick(ModulePage.upgrade_button));
```
Comment (SimonGrn) [inline tests/E2E/test/campaigns/install_upgrade/02_autoupgrade.js:71]: How is that block of code less prone to errors than scrolling to the upgrade block before clicking ?
We need to be sure that it works in a small screen too.

**Review by SimonGrn — COMMENTED** [review]
Thanks @nesrineabdmouleh

File: tests/E2E/test/campaigns/install_upgrade/02_autoupgrade.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/E2E/test/campaigns/install_upgrade/02_autoupgrade.js:71]: Yes, I tested with the small screen and now it works fine, the test is red with the scrollTo that's why I delete the line.
And also there is a mistake in `waitForExistAndClick` there is a margin of 50 in the place of the timeout

**Comment by eternoendless** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**